pkg/katautils/config.go

```diff
@@ -762,6 +807,7 @@ func initConfig() (config oci.RuntimeConfig, err error) {
 		DefaultMaxVCPUs:         defaultMaxVCPUCount,
 		MemorySize:              defaultMemSize,
 		MemOffset:               defaultMemOffset,
+		DisableBlockDeviceUse:   defaultDisableBlockDeviceUse,
```

Comment:
@jodh-intel what / why?

Reply:
If you look at `pkg/katautils/config-settings.go`, the default value for `defaultDisableBlockDeviceUse == false`. This isn't a behaviour change, it's just that previously the function in that file didn't specify a value for `DisableBlockDeviceUse`, so I added it for completeness.